Tools/clinic/requirements-dev.txt

```
@@ -0,0 +1,2 @@
+# Requirements file for external linters and checks we run on Tools/clinic/ in CI
+mypy==1.3.0
```

Not necessary, but in case you want to test dependabot picks this up after merge:
```suggestion
mypy==1.2.0
```

---

will it file an insta-PR, or will it wait until June 1st, do you know?

---

I can't remember, I _think_ it will insta-PR, but that could be only when creating the file...

---

There are some docs here: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates#frequency-of-dependabot-pull-requests

They say:

> Dependabot will also run an update on subsequent changes to the configuration file.

And [these docs](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#scheduleinterval) also say:

> Note: schedule defines when Dependabot attempts a new update. However, it's not the only time you may receive pull requests. Updates can be triggered based on changes to your dependabot.yml file, changes to your manifest file(s) after a failed update, or Dependabot security updates.

So I _think_ we'll get an insta-PR. Let's find out!